projects/wp-nextjs-app/src/app/layout.tsx

```diff
@@ -16,7 +17,10 @@ const RootLayout = ({
 }>) => {
 	return (
 		<html lang="en">
-			<body className={inter.className}>{children}</body>
+			<body className={inter.className}>
+				{children}
+				<PreviewIndicator className="form-container" />
```

Comment:
Adding this to the Root Layout directly for now, at some point we might want to expose a `<HeadstartWPApp />` component that would replace `<HeadlessApp />` in pages router